vehicle/renault/provider.go

```diff
@@ -147,3 +155,29 @@ func (v *Provider) WakeUp() error {
 	_, err := v.wakeup()
 	return err
 }
+
+var _ api.VehiclePosition = (*Provider)(nil)
+
+// Position implements the api.VehiclePosition interface
+func (v *Provider) Position() (float64, float64, error) {
+	res, err := v.position()
+	if err == nil {
+		return res.Data.Attributes.Latitude, res.Data.Attributes.Longitude, nil
+	}
+
+	return 0, 0, err
+}
+
+var _ api.VehicleChargeController = (*Provider)(nil)
+
+// StartCharge implements the api.VehicleChargeController interface
+func (v *Provider) StartCharge() error {
+	_, err := v.action(kamereon.ActionStart)
+	return err
+}
+
+// StopCharge implements the api.VehicleChargeController interface
+func (v *Provider) StopCharge() error {
+	_, err := v.action(kamereon.ActionStart)
```

Comment:
Looked because of #10828 on this code.

Is it correct to call on `StopCharge` the `kamereon.ActionStart` and not `kamereon.ActionStop`?

Reply:
Certainly not :/. Thanks for submitting PR!